streisand

```diff
@@ -119,9 +119,25 @@ EOF
   echo; echo; ansible-playbook -i inventories/inventory-existing --extra-vars=@global_vars/vars.yml playbooks/existing-server.yml
 }
 
+# be_careful_friend asks you to pay attention because you're about to do
+# something that might be impossible to undo!
+function be_careful_friend() {
+  read -r -p "$1" confirm
+  case "$confirm" in
+    streisand) echo; echo "Confirmed. Continuing";;
+    *) echo; echo "Cancelling & exiting."; exit 1;;
+  esac
+}
+
+# runs the validation role to sanity check global_vars/vars.yml
```

Comment:
same nit as before RE: sanity check. how about: "runs the validation role to validate global_vars/vars.yml"